src/ui/public/agg_response/tabify/tabify.js

```diff
@@ -61,14 +63,17 @@ export function AggResponseTabifyProvider(Private, Notifier) {
         }
         break;
       case 'metrics':
-        const value = agg.getValue(bucket);
+        let value = agg.getValue(bucket);
+        if (aggScale !== 1) {
```

Comment:
I should probably put a comment here but the if statement is there to prevent "scaling" non numbers.  Since only "count" and "sum" are allowed to scale, if you pick a different metric (such as top_hits) and don't have this if statement, then it could multiple a string with 1 and get a NaN.  By only scaling if the scale has changed, it is guaranteed that the value to be scaled is a number.

Hope that makes sense :)